packages/docs-mention/package.json

```diff
@@ -0,0 +1,101 @@
+{
+    "name": "@univerjs/docs-mention",
+    "version": "0.2.3",
+    "private": false,
```

Comment:
Let's make mention plugins as private for a while, because we haven't provide the corresponding backend service yet.